app/views/layouts/base.html.erb

```diff
@@ -101,7 +101,7 @@
         :script,
         {
           'appName' => APP_NAME,
-          'analyticsEndpoint' => api_logger_path,
+          'analyticsEndpoint' => (api_logger_path if IdentityConfig.store.frontend_analytics_enabled),
```

Comment:
If `analyticsEndpoint` is not set, [`trackEvent` won't make any requests](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/analytics/index.ts#L19-L22).